ui/src/app/components/object-detail/object-detail.component.scss

```diff
@@ -54,14 +54,14 @@
   }
 }
 
-.add-index-column {
-  margin-left: 50%;
+.add-index-column-container {
```

Comment:
Css of this container needs to be modified to align with object explorer container : https://drive.google.com/file/d/1gjleCmezeelSe8_F8zsm6-UdGXTmfBMx/view?usp=sharing&resourcekey=0-rmMWyXXwPhIq1Wzc_6nJlg

Reply:
Can you please highlight in a screenshoot and also which row you're referring to? We check in figma and for index there is an empty row at the top and thats why we have removed the empty row.
<img width="671" alt="Screenshot 2022-06-27 at 12 34 26 PM" src="https://user-images.githubusercontent.com/97866959/175879813-4210fe49-b9b9-4097-9b67-bd1308a190df.png">

Reply:
I mean the alignment here : https://drive.google.com/file/d/1dOveTyMceCTBIz_KpIFO2X95lCFWhk0K/view?usp=sharing&resourcekey=0-u0zDjiI3Q4nJ8E6AbUzvTg . I see that its as per figma but can be improved for better UI if its not a big change

Reply:
Css improved.

Reply:
Resolved.